ql/src/test/results/clientpositive/druid/druidmini_expressions.q.out

```diff
@@ -1868,9 +1868,9 @@ POSTHOOK: query: SELECT DATE_ADD(cast(`__time` as date), CAST((cdouble / 1000) A
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@druid_table_alltypesorc
 POSTHOOK: Output: hdfs://### HDFS PATH ###
-1969-02-26	1970-11-04
-1969-03-19	1970-10-14
-1969-11-13	1970-02-17
+1969-12-15	1970-01-16
+1969-12-15	1970-01-16
+1969-12-15	1970-01-16
```

Comment:
same here seems like timezone issue

Reply:
checked it out, DATE_ADD(cast(__time as date), CAST((cdouble / 1000) changed as the cdouble value changed due to change in rollup.
Added more columns to this query to make things more clear in this PR